### PR TITLE
Closes #3441 deprecate import_export_test.py

### LINKED ISSUE #3441: deprecate import_export_test.py

### PR DESCRIPTION
**Pull request by ajpotts**
None of the tests from this module were in PROTO_tests, so I had to move the entire file and just did a minimal refactor so the tests would run.

Because they share a temp directory, I had to refactor io_test.py a bit.

Closes #3441 deprecate import_export_test.py

### CODE REVIEW

File: PROTO_tests/tests/import_export_test.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    def finalizer():
+        # Clean up any resources if needed
+        io_util.delete_directory(ie_test_base_tmp)
+
+    # Register the finalizer to ensure cleanup
+    request.addfinalizer(finalizer)
+    return ie_test_base_tmp
+
+
+class TestImportExport:
```
Comment (stress-tess) [inline PROTO_tests/tests/import_export_test.py:28]: I thought these were covered in `io_test` (we tried to refactor to have all the io tests in the same file using class names to differentiate them)

File: PROTO_tests/tests/io_test.py [hunk 2]
```diff
@@ -1523,8 +1575,6 @@ def test_csv_read_write(self):
 
 
 class TestImportExport:
```
Comment (stress-tess) [inline PROTO_tests/tests/io_test.py:1577]: do this tests in this class cover the tests from `import_export_test.py`? or do they need to be updated

Comment (ajpotts) [reply PROTO_tests/tests/io_test.py:1726]: Thanks for pointing this out, I deleted the `import_export_test.py` file.

File: PROTO_tests/tests/import_export_test.py (hunk 1 above)
Comment (ajpotts) [reply PROTO_tests/tests/import_export_test.py]: Removed :)

**Review by stress-tess — APPROVED** [review]
looks great!

**Pull request merged**